lxd/db/generate/db/parse.go

```diff
@@ -139,12 +139,48 @@ func Parse(pkg *ast.Package, name string, kind string) (*Mapping, error) {
 		return nil, errors.Wrapf(err, "Failed to parse %q", name)
 	}
 
+	filterStr := findStruct(pkg.Scope, name+"Filter")
```

Comment:
@masnax to aid in understanding and readability it would be worth adding a comment to the start of this section explaining what the Filter field that is being filled here actually is and why it is being done. This also helps when reviewing to understand what the author is aiming to do.